store/src/java/com/zimbra/cs/mailbox/CalendarItem.java

```diff
@@ -736,7 +736,8 @@ private boolean updateRecurrence(long nextAlarm) throws ServiceException {
                             ((Recurrence.RecurrenceRule) mRecurrence).addException(cancelRule);
                         }
                     } else if (method.equals(ICalTok.REQUEST.toString()) ||
-                        method.equals(ICalTok.PUBLISH.toString())) {
+                        method.equals(ICalTok.PUBLISH.toString()) ||
+                        method.equals(ICalTok.REPLY.toString())) {
```

Comment:
@preshitaagrawal this looks good, but have you tested this change with other clients to be sure its all good there too?

Reply:
yes, did sanity around thunderbird and EWS clients and also tested from ZWC, it looks fine.